frontend-react/src/styles/_components.scss

```diff
@@ -11,6 +11,55 @@ hr {
     padding: 0.25rem 0.25rem;
 }
 
+// MessageReceivers.tsx
+.message-receivers-table {
+  .usa-table thead th {
+    cursor: pointer;
+    user-select: none;
+    background-color: #f0f0f0;
```

Comment:
Can this use `color('base-lightest')` instead?

Reply:
Will use color token `gray-5`